packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -76,6 +80,7 @@ export const tide = (axios, site, config) => ({
       timeout: 9000,
       headers: {
         'Content-Type': 'application/vnd.api+json;charset=UTF-8',
+        'X-Sdp-Request-Location': 'tide',
```

Comment:
If we're building with ES6 and Babel can we do?

```
headers: {
  [RPL_HEADER.REQ_LOCATION]: 'tide'
}
```

Reply:
Hey @steveworley yes, I missed it, will fix it.